requirements-dev.txt

```diff
@@ -167,7 +167,7 @@ typing-extensions==4.5.0
     #   -c requirements.txt
     #   black
     #   platformdirs
-urllib3==1.26.6
+urllib3==1.26.18
```

Comment:
This change (commit 0db6b4db) should be squashed into the same commit that changes the version in `requirements.txt` (b7253d8f2).

Reply:
Will squash